tasks/xpack/security/elasticsearch-security.yml

```diff
@@ -8,13 +8,13 @@
   file: path={{ conf_dir }}{{ es_xpack_conf_subdir }} state=directory owner={{ es_user }} group={{ es_group }}
   changed_when: False
   when:
-    - es_enable_xpack and "security" in es_xpack_features
+    - es_enable_xpack and es_xpack_features is defined and "security" in es_xpack_features
```

Comment:
These when conditionals could be moved into the main playbook. Since there is no point calling the elasticsearch-security.yml file if it isn't enabled.